<dec f='qv2ray/test/catch.hpp' l='5393' type='void Catch::AssertionResult::AssertionResult(const Catch::AssertionInfo &amp; info, const Catch::AssertionResultData &amp; data)'/>
<def f='qv2ray/test/catch.hpp' l='8299' ll='8302' type='void Catch::AssertionResult::AssertionResult(const Catch::AssertionInfo &amp; info, const Catch::AssertionResultData &amp; data)'/>
<use f='qv2ray/test/catch.hpp' l='12918' u='c' c='_ZN5Catch10RunContext25handleFatalErrorConditionENS_9StringRefE'/>
<use f='qv2ray/test/catch.hpp' l='13062' u='c' c='_ZN5Catch10RunContext10reportExprERKNS_13AssertionInfoENS_9ResultWas6OfTypeEPKNS_20ITransientExpressionEb'/>
<use f='qv2ray/test/catch.hpp' l='13080' u='c' c='_ZN5Catch10RunContext13handleMessageERKNS_13AssertionInfoENS_9ResultWas6OfTypeERKNS_9StringRefERNS_17AssertionReactionE'/>
<use f='qv2ray/test/catch.hpp' l='13101' u='c' c='_ZN5Catch10RunContext33handleUnexpectedInflightExceptionERKNS_13AssertionInfoERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEERNS_17AssertionReactionE'/>
<use f='qv2ray/test/catch.hpp' l='13118' u='c' c='_ZN5Catch10RunContext16handleIncompleteERKNS_13AssertionInfoE'/>
<use f='qv2ray/test/catch.hpp' l='13129' u='c' c='_ZN5Catch10RunContext13handleNonExprERKNS_13AssertionInfoENS_9ResultWas6OfTypeERNS_17AssertionReactionE'/>
